.github/workflows/go.yml

```diff
@@ -15,32 +15,56 @@ jobs:
     name: Run Go benchmark example
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-go@v4
+      - name: Checkout main repository
```

Comment:
This is a comparison result between the previous(910ba1f61907c6265d5401ba5740a8bc23961c8d) and the current commit(aab1e3758a7a4fd911155bc046c9743b3269920b).